notebooks/pregnancy/maternal_brain_project/r2_pvalues_all_enigma_meshes_new.ipynb

```diff
@@ -0,0 +1,1445 @@
+{
```

Comment:
<p>I can understand obj_regr versus regr really well: brilliant code design!</p><p> </p><p>can we also get the mean value of the MSE, R2 across all pcs? </p><p> </p><p>is res_val residual variance, wht's the formula for this one? std_err is on the regr coeffs? (I think so.).</p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geometric-intelligence/herbrain/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>